R/symptome_compartment_class.R

```diff
@@ -0,0 +1,226 @@
+#' An SEIR model for compartments of different symptoms
+#'
+#' Note:
+#' 1) Sum of poulation equals to 1 at any time point.
+#' An error will be reported the sum of initial population is not equal to 1.
+#' 2) At this stage, we only consider disease-caused death cases
+#' calculating mortality.
+#'
+#' model structure:
+#'    ---------------------------(omega)---------------------------------
+#'    |                                                                 |
+#'    |                                                                 |
+#'    --> S --(lam)--> E --(gamma) ------(1-ihr)-------> I ---(nu.i)--> R <--
+#'                                 |                     |                  |
+#'                                 |                 (pdeath.i)             |
+#'                                 |                     |                  |
+#'                                 |            --> CM <--                  |
+#'                                 |            |                           |
+#'                                 |      (pdeath.hsa)                      |
+#'                                 |            |                           |
+#'                                 -- (ihr)---> HA --(nu.hsa)----------------
+
+#%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+# Class definitions
+#%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+#' @describeIn model_symptome to set up dependent classes for model_symptome
+setClass("ggplot")
+setClass("gg")
+
+#' An S4 class of an basic SEIR model with compartments of different symptomes
```

Comment:
"symptoms" -- I think you can be a bit more specific here about classes with different symptoms. Is this the place to write down the model?